x-pack/metricbeat/module/sql/query/query_integration_test.go

```diff
@@ -178,10 +183,44 @@ func TestPostgreSQL(t *testing.T) {
 				Assertion: func(t *testing.T, event beat.Event) {
 					value, err := event.GetValue("sql.query")
 					assert.NoError(t, err)
-					require.NotEmpty(t, value.(map[string]interface{}))
+					require.NotEmpty(t, value)
+				},
+			}
+
+			// Need to add this.
```

Comment:
Also here, please explain why this is needed, or remove the comment if no explanation is needed.

Reply:
Removed